examples/heart_rate_monitor/main.c

```diff
@@ -265,7 +265,7 @@ static uint32_t advertising_start()
     ble_gap_adv_params_t adv_params;
 
     adv_params.type        = BLE_GAP_ADV_TYPE_ADV_IND;
-    adv_params.p_peer_addr = NULL;                           // Undirected advertisement.
+    adv_params.p_peer_addr = NULL;                        // Undirected advertisement.
```

Comment:
Are you thinking of: https://github.com/NordicSemiconductor/pc-ble-driver/blob/master/examples/heart_rate_monitor/main.c#L222? This parameter is available in `sd_api_v2/.../ble_gap.h` but it is not in `sd_api_v3/`. But is already taken care of in example. Example works for both v2 and v3. All other params are the same for`ble_gap_adv_params_t` in v2 and v3.